agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -202,6 +202,8 @@ public static class SamplingPreview {
     // future goal: make parentBased sampling the default if item count is received via tracestate
     public boolean parentBased;
 
+    public boolean suppressIngestionSampling;
```

Comment:
can we rename it to disableIngestionSampling? we use enable/disable in our json config. it's better to be consistent.

Reply:
renamed to `ingestionSamplingEnabled` (with default value true)